datacube/ui/click.py

```diff
@@ -206,9 +206,12 @@ def parse_endpoint(value):
 EXECUTOR_TYPES = {
     'serial': lambda _: get_executor(None, None),
     'multiproc': lambda workers: get_executor(None, int(workers)),
-    'distributed': lambda addr: get_executor(parse_endpoint(addr), True)
+    'distributed': lambda addr: get_executor(parse_endpoint(addr), True),
+    'celery': lambda addr: mk_celery_executor(*parse_endpoint(addr))
 }
 
+EXECUTOR_TYPES['dask'] = EXECUTOR_TYPES['distributed']  # Add alias "dask" for distributed
```

Comment:
This could make things more confusing, as dask also has synchronous, multi-threaded and multi-process schedulers/executors.